packages/extensions/venia-sample-backends/intercept.js

```diff
@@ -0,0 +1,80 @@
+const fetch = require('node-fetch');
+
+const isBackendActive = async env => {
+    try {
+        const magentoBackend = env.MAGENTO_BACKEND_URL;
+        const res = await fetch(magentoBackend);
+
+        return res.ok;
+    } catch (err) {
+        console.error(err);
```

Comment:
Similarly, I'm not sure it makes sense to log this to the user. Can we make these debug level messages too?

Reply:
You are right, these need to be debug messages. Nice suggestion @tjwiebell 👍 .